### ISSUE TITLE
no eth0 !! 

### ISSUE THREAD

**Issue by Liambeguin** [Dronolab/meta-drono#8]
eth0 not enabled by default ! 

````
root@raspberrypi2:~# ifconfig
lo        Link encap:Local Loopback
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:320 errors:0 dropped:0 overruns:0 frame:0
          TX packets:320 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0
          RX bytes:24320 (23.7 KiB)  TX bytes:24320 (23.7 KiB)
```

**Comment by Liambeguin** [comment]
```
[    3.390468] smsc95xx 1-1.1:1.0 eth0: register 'smsc95xx' at usb-bcm2708_usb-1.1, smsc95xx USB 2.0 Ethernet, b8:27:eb:44:cd:3c
```
bootlog message about ethernet

**Comment by Liambeguin** [comment]
all good... `ifconfig eth0 192.168.xxx.yyy up` starts the interface
I just have to figure out why */etc/network/interfaces* is not created

**Comment by Liambeguin** [comment]
dhcp client does not seem to be called during the boot process !

**Comment by Liambeguin** [comment]
fixed in this commit : a34de9235114b9f601567d6d668c611ea677870c
